File: backend/exchange/src/main/java/ar/com/tandilweb/exchange/userAuth/BackwardValidation.java
package ar.com.tandilweb.exchange.userAuth;

import ar.com.tandilweb.exchange.UserAuthSchema;
import ar.com.tandilweb.exchange.userAuth.types.ChallengeActions;

public class BackwardValidation extends UserAuthSchema {
	
	public long idChallenge;
	public ChallengeActions action;
	
	public BackwardValidation() {
		super("backwardValidation");
	}

}

File: frontend/newClient/src/app/services/network/epprProtocol/userAuth/BackwardValidation.ts
import { UserAuthSchema } from '../UserAuthSchema';
import { ChallengeActions } from './types/ChallengeActions';

export class BackwardValidation extends UserAuthSchema {

    public idChallenge: number;
    public action: ChallengeActions;

    constructor() {
        super('backwardValidation');
    }

}